contracts/teller-network/Escrow.sol

```diff
@@ -223,15 +301,17 @@ contract Escrow is Pausable, MessageSigned {
      * @param trx EscrowTransaction with details of transaction to be marked as canceled
      */
     function _cancel(uint _escrowId, EscrowTransaction storage trx) private {
-        trx.canceled = true;
-
-        if(trx.token == address(0)){
-            trx.seller.transfer(trx.amount);
-        } else {
-            ERC20Token token = ERC20Token(trx.token);
-            require(token.transfer(trx.seller, trx.amount));
+        if(trx.status == EscrowStatus.FUNDED){
```

Comment:
There is probably some state where it is not possible to cancel?

Reply:
I imagine a cancel could happen only for a `CREATED` or `FUNDED` state, with `FUNDED` being the only state where there are eth/tokens stored in the contract.

Reply:
An additional change that came from this observation is that a seller can now call `fund` any number of times to keep adding crypto to the tokens while the escrow status is CREATED or FUNDED.

Do you think the seller should be able to keep funding the escrow in any status different from CANCELED or RELEASED?